### ISSUE TITLE
show progress on rsync

### ISSUE THREAD

**Issue by sandover** [Oblong/obi#52]

**Comment by sandover** [comment]
Is this now closed @jshrake ?

**Comment by jshrake** [comment]
No. The above commit was optimistic, but unfortunately, not a fix. If I recall, commit f024e9a4c1f is the cause of this issue.

**Comment by tomjakubowski** [comment]
How do you show multi-destination rsync progress, anyway?

**Comment by sandover** [comment]
If there is a mode where rsync will print out progress in lines, instead of as a progress bar (`pv` has an option like this), then you could conceivably interleave them.  It wouldn't be perfect but it would be some info.
